### PR TITLE
feat(actions/plan): add argument to suppress PR comments.

### PR DESCRIPTION
**Pull request by neverendingqs**
The plan is available in the job summary (https://github.com/Brightspace/terraform-workflows/pull/18). Thoughts on adding an arg to suppress PR comments?

### CODE REVIEW

**Review by omsmith — APPROVED** [review]
Could you update the shared workflow as well? https://github.com/Brightspace/terraform-workflows/blob/main/.github/workflows/workflow.yml

**Comment by neverendingqs** [comment]
> Could you update the shared workflow as well? https://github.com/Brightspace/terraform-workflows/blob/main/.github/workflows/workflow.yml

:+1:. 03a385ddecd1a27c518102afed302c0b39414269. Will try out the action on the repo I was thinking of applying this on.

Note: I didn't investigate, but the editor was advising something was wrong with `matrix`: 
image

Seems like editor might be the one that's wrong?

**Review by omsmith — APPROVED** [review]

**Comment by omsmith** [comment]
> Seems like editor might be the one that's wrong?

They updated the syntax, but still allow strings - the editor is preferring the newer syntax. S'all good.

File: actions/plan/action.yml [hunk 1]
```diff
@@ -67,6 +72,7 @@ runs:
         PATH_TO_ARCHIVE: ${{ steps.setup.outputs.artifacts_dir }}
 
     - working-directory: ${{ fromJson(inputs.config).workspace_path }}
+      if: ${{ inputs.comment_plan == true }}
```
Comment (neverendingqs) [inline actions/plan/action.yml:75]: I'm out of ideas atm. I've tried this (doesn't work for `true`) and 

```suggestion
      if: ${{ inputs.comment_plan }}  # doesn't work for `false`
```

```suggestion
      if: inputs.comment_plan  # doesn't work for false
```

Maybe related to https://github.com/actions/runner/issues/1173?

PR used to test, although it's hard to line up the commits here and there: https://github.com/Brightspace/d2l-eventwebhooks/pull/124

Comment (neverendingqs) [reply actions/plan/action.yml]: 6aa6137a4ff2f28d05d6c5d4cd30353a3bdc1c88 did the trick. Thanks! (See https://github.com/Brightspace/d2l-eventwebhooks/pull/124#commits-pushed-9217668)

**Pull request merged**